Demo/src/main/java/com/paypal/android/ui/card/validation/CardType.kt

```diff
@@ -0,0 +1,30 @@
+package com.paypal.android.ui.card.validation
+
+@Suppress("MagicNumber")
+enum class CardType(
+    val cardNumberIndices: List<Int>,
+    val maxCardLength: Int
+) {
+    AMERICAN_EXPRESS(
+        cardNumberIndices = listOf(4, 11),
+        maxCardLength = 15
+    ),
+    VISA(
+        cardNumberIndices = listOf(4, 9, 14),
+        maxCardLength = 16
+    ),
+    UNKNOWN_CARD(
+        cardNumberIndices = listOf(4, 9, 14),
+        maxCardLength = 16
+    )
+}
+
+fun getCardType(cardNumber: String): CardType {
+    return when {
+        cardNumber.startsWith("34") || cardNumber.startsWith("37") -> {
```

Comment:
This might get a little unruly since there are so many different number patterns for some card types. I think it's ok to leave it like this while it's just for the demo app but something to think about if we use this for determining card types in the SDK.

Reply:
Agreed! I was thinking of moving these IIN ranges into the `CardType` enum and having a function live within each card type, but thought that this was sufficient for the demo app.